README.rst

```diff
@@ -2,25 +2,18 @@ Oklahoma State University (OSU) standard cells for GF180MCU
 ===========================================================
 
 This repository will contain a new set of standard cells for the
-`GF180MCU process technology release by Google and GlobalFoundries <https://github.com/google/gf180mcu>`_.
+`GF180MCU process technology release by Google and GlobalFoundries
```

Comment:
I think it'd be better to link to the main repo.

Reply:
done...thx!